content/en/community.md

```diff
@@ -3,28 +3,77 @@ title: Community
+We offer several communication channels to learn, share your knowledge and connect with others within the NumPy community.
 
-### Participate online
+The NumPy leadership has made a strong commitment to creating an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/code-of-conduct.md) for guidance on how to interact with others in a way that makes the community thrive.
 
-NumPy also has a number of online communication channels to help keep in touch.
+## Online Communities and Social Resources
 
-As you interact with others in the NumPy community, remember that we have a strong commitment to being an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/devdocs/dev/conduct/code_of_conduct.html) for guidance on how to interact with others in a way that makes the community thrive.
+### [NumPy mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion)
+for longer-form discussions, e.g.:
 
-Below is a short list of communication channels and GitHub repositories where you can get involved. We always welcome participation in the NumPy community.
+   - how should we have discussions about NumPy as software and as a community project?
 
-TODO:
+   - developing or adding items to the NumPy Roadmap;
+
+   - announcing sprints or conference talks;
+
+   - changes to the development workflow;
+
+   - announcements for NumPy releases and developer meetings.
+   
+
+### [StackOverflow](http://stackoverflow.com/questions/tagged/numpy) 
+
+for *usage* questions, e.g.  "How do I do *X* in NumPy?” Please use the ‘#numpy tag’.
+
+
+### [Reddit](https://www.reddit.com/r/Numpy/) 
+
+for *usage* questions.
+
+
+### [GitHub issue tracker](https://github.com/numpy/numpy/issues)
+
+   - **bug reports** (e.g. "``np.arange(3).shape`` returns ``(5,)``, when it should return ``(3,)``.");
+
+   - **documentation issues** (e.g. "I found this section unclear");
+
+   - **feature requests** (e.g. "I would like to have a new interpolation method in ``np.percentile``.").
+
+
+### [Gitter](https://gitter.im/numpy/numpy)
+
+ a real-time chat room where users and community members help each other.
+
+
+### [Slack](https://app.slack.com/client/TM52KRJ2K/CLRQESU4T)
+
+a real-time chat room to ask questions about contributing to NumPy.
+
+If you’ve found a security vulnerability in NumPy, please report it using the procedure documented at[ https://tidelift.com/docs/security](https://tidelift.com/docs/security)
+
+
+## Study Groups and Meetups
+
+[PyData](https://www.meetup.com/pro/pydata/)
+
+
+## Conferences and Events
```

Comment:
Also useful. I'd add the other SciPy conferences, and a note on sprints and where they get announced.